.env.generic

```diff
@@ -1,5 +1,6 @@
 PROGRAMMES_ENABLED=HPV,MENACWY,TD_IPV,FLU
 BASE_URL=https://qa.mavistesting.com
+JIRA_URL=
```

Comment:
I'm wondering whether we can make this hard coded, or at least have a default, as it's unlikely we'll have multiple Jira instances.